gtsfm/frontend/detector_descriptor/aslfeat.py

```diff
@@ -0,0 +1,95 @@
+"""Implements the ASLFeat detector-descriptor by wrapping around the original Pytorch source code.
+
+Paper: https://arxiv.org/abs/1905.03561
+https://github.com/mihaidusmanu/d2-net
+
+Authors: Travis Driver
+"""
+
+from pathlib import Path
+from typing import Tuple, Dict, Any
+import sys
+
+import numpy as np
+import tensorflow.compat.v1 as tf
+
+import gtsfm.utils.images as image_utils
+from gtsfm.common.image import Image
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.detector_descriptor.detector_descriptor_base import DetectorDescriptorBase
+
+sys.path.append("thirdparty/ASLFeat")
```

Comment:
do we need this? are the thirdparty modules not added without it?

Reply:
Yes, the ASLFeat library uses relative paths in the code. Although, I could fix these in the ASLFeat code since all of the files have been checked in?